cmd/minikube/cmd/start_flags.go

```diff
@@ -142,6 +142,7 @@ const (
 	socketVMnetPath         = "socket-vmnet-path"
 	staticIP                = "static-ip"
 	autoPauseInterval       = "auto-pause-interval"
+	enableNvidiaGPUs        = "enable-nvidia-gpus"
```

Comment:
seems to long word, I rather something shorter the user can type easier
how about --enable-nvidia

Reply:
I think having something about GPUs would make it clear that it's GPU related. Especially if we add support for AMD, AMD makes CPUs and GPUs, `--enable-amd` would be broad and possibly confusing. `--nvidia-gpus` maybe?

Reply:
how about --gpus
with options such as 
nvidia, amd and all (if later we support both at same time)
and gives us room so in the future if we wanna have it Docker Style so the user can specify their own device

```
docker run --gpus device=GPU-3a23c669-1f69-c64e-cf85-44e9b07e7a2a
```

that would make users learn the syntax easier since it is already used in docker

https://docs.docker.com/engine/reference/commandline/run/#gpus

Reply:
Renamed the `enable-nvidia-gpus` flag to `gpus` and made it a string flag that accepts `all` & `nvidia`.